### ISSUE TITLE
Create FAQ

### ISSUE THREAD

**Issue by Pol-Lanski** [dappnode/DAppNodeDocs#166]
Compile a list of the most frequent questions from users.
Create a FAQ addressing these issues.

**Comment by tropicar** [comment]
Currently, we have 2 important places where there are solutions to the problem that a user can find:
- https://docs.dappnode.io/user-guide/support/faqs
- https://docs.dappnode.io/user-guide/support/troubleshooting

Firstly, there is a lot of stuff that is very old. Furthermore, it would be interesting to join these two parts. Not to have troubleshooting and FAQ.

**Comment by tropicar** [comment]
I will collect here all the FAQS that i have thought and people have suggested:
Here there is a collection of so many faqs people used to do, i will separate in 2 main parts: support part and general. This list is not finished.
List of FAQS:
**General**
- What is DAppNode?
- What is the point of DAppNode?
- What can I do with DAppNode?
- What do I need to run a DAppNode?
- What are the advantages of running a DAppNode?
- What is a DAO? What is de DAppNode DAO?
- Why is it "good" having your own nodes?
- I can't access to my DAppNode

Users suggestions:
Do I need a monitor and a keyboard for my Dappnode?
What are the first steps after I logged in? (change password, create vpn,�)
Which packages do I need to install?
Which packages should I install? (fallback, dms,�?)
How do I install a fallback?
Is there a way to monitor CPU usage or validator uptime? (Grafana)
How to test staking with goEth and where do I get goEth?
Why is my Dappnode louder than normal?
Is there a way to make it completely silent even after syncing? (fanless case, or set bios to 15W pl1, 19W pl2 with [75, 25%, 2] Fan settings)
What can I do if my harddrive runs out of space? (insert another drive?)
Who helps me if I run into a problem? (Discord Dappnode, EthStaker, Mentorprogram,�?)
Should I set up an UPS ?
Is my normal router secure and does it fit the needs of staking with a DappNode?
What are the most important security aspects? (generate keys offline, secure passwords, mnemonic on metal seedphrase storage, Firewall)
What is UPnP?
What is Portforwarding and why do I need it?
How to find out which Ports I need to forward?
What happens if my node is offline while validating - do I get slashed?
Is my internet connection fast enough?
Do I need to type in commands on a terminal?
What is EthDocker?
What is a static IP and why do I need it?
How much time do I need to set all things up?
How much time do I need to keep everything running?
Where do I find guidelines to set up my validator and how do I know if they are not outdated?
